src/main/java/chatcat/tasklist/TaskList.java

```diff
@@ -0,0 +1,212 @@
+package chatcat.tasklist;
+
+import java.util.ArrayList;
+
+import chatcat.tasks.Deadline;
+import chatcat.tasks.Event;
+import chatcat.tasks.Task;
+import chatcat.tasks.Todo;
+import chatcat.util.WriteToFile;
+import chatcat.util.DateTimeUtil;
+import chatcat.chatcatexception.ChatCatException;
+import chatcat.commands.Commands;
+
+/**
+ * Handles writing and reading to list of tasks represented as an {@code ArrayList}.
+ * Allows clients to unmark, mark, add and delete tasks. Allows users to view task list.
+ */
+public class TaskList {
+    ArrayList<Task> Tasks = new ArrayList<>();
```

Comment:
```suggestion
    ArrayList<Task> tasks = new ArrayList<>();
```
Great job naming the arraylist in plural form since its a list of tasks but variables should not start with a capitalization.